backend/attachRendererFiber.js

```diff
@@ -23,51 +23,86 @@ function getInternalReactConstants(version) {
   var ReactTypeOfSideEffect;
 
   // **********************************************************
-  // The section below is copy-pasted from files in React repo.
+  // The section below is copied from files in React repo.
   // Keep it in sync, and add version guards if it changes.
   // **********************************************************
-  if (semver.gte(version, '16.4.3-alpha')) {
+  if (semver.gte(version, '16.6.0-beta.0')) {
     ReactTypeOfWork = {
+      ClassComponent: 1,
+      ClassComponentLazy: -1, // Removed
+      ContextConsumer: 9,
+      ContextProvider: 10,
+      CoroutineComponent: -1, // Removed
+      CoroutineHandlerPhase: -1, // Removed
+      ForwardRef: 11,
+      ForwardRefLazy: -1, // Removed
+      Fragment: 7,
       FunctionalComponent: 0,
-      FunctionalComponentLazy: 1,
+      FunctionalComponentLazy: -1, // Removed
+      HostComponent: 5,
+      HostPortal: 4,
+      HostRoot: 3,
+      HostText: 6,
+      IndeterminateComponent: 2,
+      LazyComponent: 16,
+      MemoComponent: 14,
+      Mode: 8,
+      Profiler: 12,
+      SimpleMemoComponent: 15,
+      SuspenseComponent: 13,
+      YieldComponent: -1, // Removed
+    };
+  } else if (semver.gte(version, '16.4.3-alpha')) {
+    ReactTypeOfWork = {
       ClassComponent: 2,
       ClassComponentLazy: 3,
```

Comment:
We didn't actually release these in public releases since they were behind the suspense flag. It seems like we can drop support for this tag now.  I don't think RN is using lazy yet neither and if when they start, we'll be on 16.6 anyway. We don't support alphas continuously neither.

Reply:
Ah! I didn't know that the *Lazy tags never went out in a non-alpha. That's great, I'll remove them.